Sources/AIProxy/OpenAI/OpenAIRealtimeResponseFunctionCallArgumentsDone.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  OpenAIRealtimeResponseDone.swift
+//  AIProxy
+//
+//  Created by Tim Wheeler on 4/22/25.
+//
+
+import Foundation
+
+public struct OpenAIRealtimeResponseFunctionCallArgumentsDone: Encodable {
+    public let type = "response.function_call_arguments.done"
+    public let response: Response?
+
+    public init(response: Response? = nil) {
+        self.response = response
+    }
+}
+
+// MARK: -
+extension OpenAIRealtimeResponseFunctionCallArgumentsDone {
+    
+    public struct Response: Encodable {
+        public let name: String?
+        public let arguments: String?
```

Comment:
I didn't include all property payloads such as `eventId` etc., which follows existing patterns I've seen in this codebase.